### PR TITLE
Updated User Guide

### PR DESCRIPTION
**Pull request by triishaa**

### CODE REVIEW

**Review by OrangeCatLoves — APPROVED** [review]
LGTM!

File: docs/images/addName.png
Comment (Potatoad88) [inline docs/images/addName.png:1]: Should it be add only instead of addName?

**Review by Potatoad88 — APPROVED** [review]
LGTM except maybe the naming of the add command?

File: docs/images/addName.png
Comment (triishaa) [reply docs/images/addName.png:1]: will edit it in the next pull request

**Pull request merged**